### DISCUSSION TITLE
Microsoft Clarity

### DISCUSSION THREAD

**Discussion by borrmann** [dotnet/maui#21941] [Q&A]
Is there a way to integrate MS Clarity with MAUI?

**Comment by davidortinau** [comment]
They have an Android SDK you could bind, perhaps using [slim bindings](https://github.com/Redth/DotNet.Platform.SlimBindings). 

https://learn.microsoft.com/en-us/clarity/mobile-sdk/android-sdk
https://mvnrepository.com/artifact/com.microsoft.clarity/clarity

I don't see iOS, Mac, or Windows SDKs, so not sure what you'd do there.

I think that's an uphill battle, and there's probably existing .NET supported services to accomplish similar.

**Reply by borrmann** [reply]
Thanks!
The slim bindings approach looks promising once other platforms are added by MS clarity.
In the MS clarity GitHub I saw they're currently working on getting iOS ready too.

**Comment by Kebechet** [comment]
I have wrapped clarity into MAUI. So you can take a look at released [nuget package](https://www.nuget.org/packages/Kebechet.Maui.MicrosoftClarity/) or the [maui repo](https://github.com/Kebechet/Maui.MicrosoftClarity)
